Library/Homebrew/extend/os/linux/install.rb

```diff
@@ -14,6 +14,19 @@ module Install
       "/system/bin/linker",
     ].freeze
 
+    def check_cpu
+      return if (Hardware::CPU.intel? && Hardware::CPU.is_64_bit? || Hardware::CPU.arm?)
```

Comment:
```suggestion
      return if (Hardware::CPU.intel? && Hardware::CPU.is_64_bit?) || Hardware::CPU.arm?
```

My bad on style here sorry!